src/Chartjs/assets/dist/controller.js

```diff
@@ -1,6 +1,7 @@
 import { Controller } from '@hotwired/stimulus';
-import Chart from 'chart.js/auto';
+import { Chart, registerables } from 'chart.js';
 
+Chart.register(...registerables);
```

Comment:
```suggestion
registerables.forEach(registerable => Chart.register(registerable));
```
https://stackoverflow.com/questions/75449457/typeerror-chart-registerables-is-not-iterable-cannot-read-property-undefined

Reply:
I came across this one too, however we still need to verfify `registerables != undefined` as with this code we would still get the following error on `3.x`:

```js
TypeError: Cannot read properties of undefined (reading 'forEach')
```

Reply:
And i still think it should not be "undefined" :))

Reply:
https://github.com/evertharmeling/ux-1263 

This seems to backup that it is our testsuite that's messing with us. I've added a console log on the `registerables` ([symfony/ux](https://github.com/evertharmeling/ux/commit/419f553fd985a8491ceb93ee5f5c09088a2c0dfb)). And when using `chart.js@3.9.1` the `registerables` variable is NOT `undefined`.

Reply:
Ok so its a « module » thing we need to force on the tests… that narrows the problem !

thanks a lot for taking this time it really helps a lot :)